src/ListOptions.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\Sandboxes\Api;
+
+class ListOptions
+{
+    private ?string $branchId = null;
+
+    public static function create(): self
+    {
+        return new self();
+    }
```

Comment:
just curious why static and not constructor

Reply:
Just because I'm lazy typing full `(new ListOptions())->setBranchId()` :smile: You have to do `new ListOptions`, manually wrap it into brackets and now IDE suggests you the `setBranchId` call. When written as `ListOptions::create()->setBranchId()` you write just the class name and IDE than automatically suggests possible `::create()` call, then you put `->` and `setBranchId()` is suggested.

Reply:
alright, I think it's probably fine :-)